db/crud_test.go

```diff
@@ -739,3 +741,193 @@ func TestMalformedRevisionStorageRecovery(t *testing.T) {
 	_, err := db.PutExistingRevWithBody("doc1", rev3c_body, []string{"3-c", "2-b", "1-a"}, false)
 	assert.NoError(t, err, "add 3-c")
 }
+
+func BenchmarkDatabaseGet1xRev(b *testing.B) {
+	db, testBucket := setupTestDB(b)
```

Comment:
Disable test logging for benchmarks so the output can be piped into benchcmp, etc. without modification.

```suggestion

	defer base.DisableTestLogging()()

	db, testBucket := setupTestDB(b)
```